client/apps/edit/components/section_artworks/index.coffee

```diff
@@ -27,13 +27,35 @@ module.exports = React.createClass
     @fetchArtworks ids if ids?.length
     @props.section.artworks.on 'add remove', => @forceUpdate()
     @toggleFillwidth()
+    @setupAutocomplete()
 
   componentWillUnmount: ->
     @props.section.artworks.off()
+    @autocomplete.remove()
 
   componentDidUpdate: ->
     @toggleFillwidth()
 
+  setupAutocomplete: ->
+    Autocomplete = require '../../../../components/autocomplete/index.coffee'
+    @autocomplete = new Autocomplete
+      url: "#{sd.ARTSY_URL}/api/search?q=%QUERY"
+      el: $(@refs.autocomplete.getDOMNode())
+      filter: (res) ->
+        vals = []
+        for r in res._embedded.results
+          if r.type == 'Artwork'
```

Comment:
Oh nice! I didn't even think v2's search would work here given it returns mixed results. That's exciting!